pax/trigger_plugins/SaveSignals.py

```diff
@@ -52,7 +53,7 @@ def group_signals(signals, event_ranges, signal_indices_buffer, is_in_event):
         if not in_event:
             if signal['left_time'] >= event_ranges[current_event, 0]:
                 if signal['left_time'] > event_ranges[current_event, 1]:
-                    raise ValueError("Error during signal grouping: event without signals??")
+                    raise TriggerGroupSignals("Error during signal grouping: event without signals??")
```

Comment:
I may propose some name changes (e.g. TriggerLogicError / TriggerSignalGroupingError) and add a few more custom exceptions at other points. Ok now of course.